packages/core/src/modules/proofs/protocol/v1/models/PresentationPreview.ts

```diff
@@ -135,4 +145,33 @@ export class PresentationPreview {
   public toJSON(): Record<string, unknown> {
     return JsonTransformer.toJSON(this)
   }
+
+  public static fromRecord(record: Record<string, string>, version?: ProofProtocolVersion) {
+    const attributes = Object.entries(record).map(
+      ([name, value]) =>
+        new PresentationPreviewAttribute({
+          name,
+          mimeType: 'text/plain',
```

Comment:
We lose the mime type here. 

Maybe I don't understand this function, docs would be nice, but it also seems to change the predicate.

Reply:
removed the method as it was unused